examples/specs/histogram_nonlinear.vl.json

```diff
@@ -48,7 +48,7 @@
   "title": "Distribution of Frame Render Time (ms)",
   "width": {"step": 40},
   "height": 100,
-  "mark": {"type": "bar", "cornerRadiusEnd": 0},
+  "mark": {"type": "bar", "cornerRadiusEnd": 0, "orient": "vertical"},
```

Comment:
Tried to add "binned" but failed due to https://github.com/vega/vega-lite/issues/8476.
Adding "orient" is also an alternative way to make this spec work.